router/pkg/config/config.schema.json

```diff
@@ -264,6 +264,15 @@
                   "type": "boolean",
                   "default": false,
                   "description": "Enable the watch mode. The watch mode is used to watch the execution config file for changes. If the file changes, the router will reload the execution config without downtime."
+                },
+                "watch_interval": {
+                  "type": "string",
+                  "description": "The interval at which the file is checked for changes. The period is specified as a string with a number and a unit, e.g. 10ms, 1s, 1m, 1h. The supported units are 'ms', 's', 'm', 'h'.",
+                  "default": "1s",
+                  "duration": {
+                    "minimum": "100ms",
```

Comment:
Because you said this is sensitive for the watcher to work, we should add a simple test.

Reply:
Added, and updated the schema to have no upper limit and a higher minimum (1s), and enforced `watch == true` if `watch_interval` is present at schema level, this should prevent mistakes where users set one and not the other.